ts/Core/Series/Point.ts

```diff
@@ -820,8 +820,19 @@ class Point {
 
     /**
      * Get the pixel position of the point relative to the plot area.
-     * @private
      * @function Highcharts.Point#pos
+     *
+     * @sample highcharts/point/position
+     *         Get point's position in pixels.
+     *
+     * @param {boolean} chartCoordinates
+     * Whether to return the position relative to the plot area of the chart.
```

Comment:
Actually the other way around.

"If true, the returned position is relative to the full chart area. If false, it is relative to the plot area determined by the axes."